package.json

```diff
@@ -2,7 +2,7 @@
     "name": "azure-deploy",
     "displayName": "Deploy to Azure",
     "description": "Generating CI/CD pipelines to Azure from GitHub and Azure Repo",
-    "version": "1.1.2",
+    "version": "1.1.3",
```

Comment:
I guess Vineet has already updated this version and shipped. You should not worry about version for now as we will club mulitple changes and ship one version. Just update the CHANGELOG file.